### DISCUSSION TITLE
energy error of test sets and training set

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#1252] [Q&A]
DP developers:
The test error and training error of energy are not very clear. DPMD defaults the last data set as the test set. The test error is the error between the predicted energy and the DFT energy of the test set, and the training error is the predicted energy and the DFT energy of the training set. Because the sample energy on the test set and the training set are different, and the predicted energy obtained through the training set must be different from the energy on the test set, what meaning does the predicted energy test on the test set have?
Looking forward to replying!
